### DISCUSSION TITLE
Adding chords to a show

### DISCUSSION THREAD

**Discussion by Nwachinyere** [ChurchApps/ChurchAppsSupport#310] [FreeShow] [answered]
I've been unsuccessful in adding chords to my show, there's is also no documentation that I've found on how to do this.  Can anyone provide instructions on how to do this?

**Comment by vassbo** [comment, marked as answer]
Hi, I will add documentation for this soon. 😃

In the "Edit" page, you can see a note icon in the bottom of the center content. Press this to activate chords mode (the note should become colored). Then you will see a "+" icon in the top right corner of all the textboxes, Click this to add a chord (or right click for more options.)

It is currently not implemented fully, so you might have a hard time dragging the chords around in the textbox, but I'll try to improve it sometime. 😊

If you want to show the chords in the stage view, you have to select the stage textbox, and click on the checkbox for "Chords".
Hope this helps.

**Reply by Nwachinyere** [reply]
Thank you so much.! I'm sure this feature will improve over time.  I know it's still a work in progress.  I'd like to suggest adding the number method as an alternative.  I know it's possible to do this right now with the "set custom value" option.  It should be just as easy as cycling through the letters. For instance, in the key of C instead of Db it could be a #1 or b2, or a 2 instead of D.  This method makes it easy to play chords to a song in any key.

**Reply by vassbo** [reply]
Good suggestion. I'll see what I can do. 😄
